drivers/sensor/lis2dh/lis2dh.c

```diff
@@ -9,6 +9,41 @@
 #include <init.h>
 #include <misc/byteorder.h>
 #include <misc/__assert.h>
+#include <logging/log.h>
+
+#define LOG_LEVEL CONFIG_SENSOR_LOG_LEVEL
+LOG_MODULE_REGISTER(lis2dh);
+
+static int lis2dh_bus_configure(struct device *dev)
```

Comment:
note that there is #10215 which will allow logging in headers. Is this function intended to be internal only? Previously was available through lis2dh.h.

Reply:
@nordic-doki  This was used internal to driver so I moved to .c and I didn't know about #10215.  There is no use of this function externally as well. If you want me to place under .h I don't mind. 
Addressed all comments from you. Please go through PR once

Reply:
did you push? The PR still has the same issues